hunt_queries/llm/rules/llm_latency_anomalies_detection.toml

```diff
@@ -0,0 +1,15 @@
+[hunt]
+author = "Elastic"
+uuid = "3708787b-811b-43b1-b2e7-c7276b8db48c"
+description = "This updated query monitors the time it takes for an LLM to start sending a response after receiving a request, focusing on the initial response latency. By calculating the difference between the expected start of the response and the recorded response time, it identifies when responses are significantly delayed. Anomalous latencies can be symptomatic of issues such as network attacks (e.g., DDoS) or system inefficiencies that need to be addressed. By tracking and analyzing latency metrics, organizations can ensure that their systems are running efficiently and securely, and can quickly respond to potential threats that might manifest as abnormal delays."
+name = "Monitoring for Latency Anomalies"
+query = '''
+from logs-aws_bedrock.invocation-*
```

Comment:
the index is quite self-explanatory but wonder if we should add which integration it uses in a dedicated field.

Reply:
Added